test/tests/ics/solution_ic/tests

```diff
@@ -13,13 +13,19 @@
     csvdiff = 'solution_scalar_ic_out.csv'
     requirement = "The system shall be able to initialize a scalar variable from an ExodusII file."
   []
+  [exodus_certain_blocks]
+    type = 'Exodiff'
+    input = 'solution_ic_block_restricted.i'
+    exodiff = 'solution_ic_block_restricted_out.e'
+    requirement = "The system shall be able to initialize a variable from an ExodusII file."
```

Comment:
This is a duplicate requirement from line 8. I'm not sure why SQA checks didn't see that